tests/test_auto/runner/test_local.py

```diff
@@ -0,0 +1,31 @@
+from functools import partial
+
+import numpy as np
+import pytest
+
+from etna.auto.runner import LocalRunner
+from etna.auto.runner import ParallelLocalRunner
+
+
+@pytest.fixture()
```

Comment:
I thought we most of the time use `@pytest.fixture` in this case and not `@pytest.fixture()`.